cmd/node/main.go

```diff
@@ -133,6 +136,18 @@ func main() {
 	}
 	addr := fmt.Sprintf(":%s", nodeConfig.ReverseProxyPort())
 
+	// Gorilla sessions persisted in the browser's cookies.
+	// TODO These need to actually be persisted somewhere
+	sessionStoreKey := make([]byte, 32)
+	_, err = rand.Read(sessionStoreKey)
+	if err != nil {
+		log.Fatal().Err(err).Msg("error generating session store key")
+	}
+	if nodeConfig.Environment() == constants.EnvironmentDev {
+		// Keep the key consistent in dev mode so that the same session is used across restarts.
+		sessionStoreKey = []byte("FaKe_DeV-SeSsIoN-KeY")
```

Comment:
can we just only have the hard coded session key for now. the random one is TODO anyway and once we have persistence we won't need to fork behaviour between dev and prod